### PR TITLE
Fix button text inconsistencies

### PR DESCRIPTION
**Pull request by savindi7**
### Purpose
Screenshot 2024-01-26 at 00 11 25

Screenshot 2024-01-26 at 00 14 30

Screenshot 2024-01-25 at 23 49 56

Screenshot 2024-01-25 at 23 50 06

### Related Issues
- https://github.com/wso2/product-is/issues/19164

### Related PRs
- Related PR `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/connections/components/create/add-connection-wizard.tsx [hunk 1]
```diff
@@ -436,10 +421,39 @@ export const CreateConnectionWizard: FC<CreateConnectionWizardPropsInterface> =
...
+                                            data-testid="add-connection-modal-next-button"
+                                            data-componentid="add-connection-modal-next-button"
+                                            loading={ isSubmitting }
+                                            disabled={ isSubmitting }
+                                        >
+                                            { t("console:develop.features.authenticationProvider." +
+                                                "wizards.buttons.next") }
+                                        </PrimaryButton>
+                                    )
+                                    : (
```
Comment (JayaShakthi97) [inline apps/console/src/features/connections/components/create/add-connection-wizard.tsx:442]:
```suggestion
                                    ) : (
```

File: apps/console/src/features/connections/components/wizards/expert-mode/expert-mode-authentication-provider-create-wizard.tsx [hunk 2]
```diff
@@ -254,33 +254,49 @@ export const ExpertModeAuthenticationProviderCreateWizard: FunctionComponent<
...
+                                                floated="right"
+                                                onClick={ () => {
+                                                    submitForm();
+                                                } }
+                                                data-componentid={ `${ componentId }-modal-finish-button` }
+                                                loading={ isSubmitting }
+                                                disabled={ isSubmitting }
+                                            >
+                                                { t("console:develop.features.authenticationProvider.wizards." +
+                                                "buttons.finish") }
```
Comment (JayaShakthi97) [inline apps/console/src/features/connections/components/wizards/expert-mode/expert-mode-authentication-provider-create-wizard.tsx:296]:
```suggestion
                                                { t("console:develop.features.authenticationProvider.wizards." +
                                                    "buttons.finish") }
```

File: apps/console/src/features/connections/components/wizards/expert-mode/expert-mode-authentication-provider-create-wizard.tsx [hunk 3]
```diff
@@ -254,33 +254,49 @@ export const ExpertModeAuthenticationProviderCreateWizard: FunctionComponent<
...
+                                                floated="right"
+                                                onClick={ () => {
+                                                    submitForm();
+                                                } }
+                                                data-componentid={ `${ componentId }-modal-finish-button` }
+                                                loading={ isSubmitting }
+                                                disabled={ isSubmitting }
+                                            >
+                                                { t("console:develop.features.authenticationProvider." +
+                                                "wizards.buttons.next") }
```
Comment (JayaShakthi97) [inline apps/console/src/features/connections/components/wizards/expert-mode/expert-mode-authentication-provider-create-wizard.tsx:283]:
```suggestion
                                                { t("console:develop.features.authenticationProvider." +
                                                    "wizards.buttons.next") }
```

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 4]
```diff
@@ -10202,7 +10202,7 @@ export const console: ConsoleNS = {
                 },
                 placeholders: {
                     emptyList: {
-                        action: "Add Organization",
+                        action: "New Organization",
```
Comment (JayaShakthi97) [inline modules/i18n/src/translations/en-US/portals/console.ts:10205]: Don't we have these translations in other languages?

Comment (savindi7) [reply modules/i18n/src/translations/en-US/portals/console.ts:10205]: fixed

**Review by JayaShakthi97 — APPROVED** [review]

**Pull request merged**